File: csharp_dotnetcore/81.Facebook-Events/FacebookModel/FacebookSender.cs
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

using Newtonsoft.Json;

namespace Facebook_Events_Bot.FacebookModel
{
    /// <summary>
    /// Defines a Facebook sender.
    /// </summary>
    public class FacebookSender
    {
        /// <summary>
        /// The Facebook Id of the sender.
        /// </summary>
        [JsonProperty("id")]
        public string Id { get; set; }
    }
}

File: samples/java_springboot/23.facebook-events/src/main/java/com/microsoft/bot/sample/facebookevents/facebookmodel/FacebookSender.java
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MT License.

package com.microsoft.bot.sample.facebookevents.facebookmodel;

import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonProperty;

/**
 * Defines a Facebook sender.
 */
public class FacebookSender {

    @JsonProperty(value = "id")
    @JsonInclude(JsonInclude.Include.NON_EMPTY)
    private String id;

    /**
     * The Facebook Id of the sender.
     * @return the Id value as a String.
     */
    public String getId() {
        return this.id;
    }

    /**
     * The Facebook Id of the sender.
     * @param withId The Id value.
     */
    public void setId(String withId) {
        this.id = withId;
    }
}
